airbyte-integrations/connector-templates/generator/plopfile.js

```diff
@@ -24,7 +24,6 @@ ${additionalMessage || ""}
 
 module.exports = function (plop) {
   const docRoot = '../../../docs/integrations';
-  const definitionRoot = '../../../airbyte-config-oss/init-oss/src/main/resources';
```

Comment:
That is the only code change, but my tooling and search told me this thing is not used.